sql/create_database.sql.in

```diff
@@ -0,0 +1,3 @@
+CREATE DATABASE czech_slovak_names 
```

Comment:
If I get it right, that should be name for whole database for RPCM? Maybe we can think about something more appropriative.

Reply:
Wau! Nice catch! This was from example :-)